src/Analyser/MutatingScope.php

```diff
@@ -365,6 +372,40 @@ public function afterClearstatcacheCall(): self
 				continue 2;
 			}
 		}
+		$nativeExpressionTypes = $this->nativeExpressionTypes;
+		foreach (array_keys($nativeExpressionTypes) as $exprString) {
+			// list from https://www.php.net/manual/en/function.clearstatcache.php
+
+			// stat(), lstat(), file_exists(), is_writable(), is_readable(), is_executable(), is_file(), is_dir(), is_link(), filectime(), fileatime(), filemtime(), fileinode(), filegroup(), fileowner(), filesize(), filetype(), and fileperms().
+			foreach ([
+						 'stat',
+						 'lstat',
```

Comment:
This is exactly the kind of duplication I'd like to avoid :)

Reply:
Definitely. I just wanted to test that it is possible to make count of native and full type same, if we add them everywhere.